lib/circuits/mdoc/circuit_maker.cc

```diff
@@ -19,29 +19,27 @@
 #include <cstddef>
 #include <cstdint>
 #include <cstdlib>
-#include <filesystem>
 #include <fstream>
 #include <iomanip>
 #include <iostream>
 #include <sstream>
 #include <string>
 #include <vector>
-#include <absl/cleanup/cleanup.h>
-#include <absl/flags/flag.h>
-#include <absl/flags/parse.h>
 
-// #include "base/init_google.h"
-// #include "file/base/path.h"
+#include "base/init_google.h"
```

Comment:
FYI this is rolling back changes from #82

Reply:
Ack